modules/props.js

```diff
@@ -49,6 +81,13 @@ module.exports = function propsModule (vnode, attributes) {
       key = 'class'
     }
 
-    attributes.set(key.toLowerCase(), escape(value))
+    var lkey = key.toLowerCase()
+    if (~booleanAttributes.indexOf(lkey)) {
+      if (value) { // set attr only when true
```

Comment:
if you really mean `true`, then I would check with `value === true`, otherwise I would say 'truthy' in the comment

```suggestion
      if (value) { // set attr only when truthy
```

Reply:
Updated!